README.md

```diff
@@ -104,68 +122,52 @@ wish to use node and npm, or are switching to a different way of managing them.
 
     n uninstall
 
-## Binary Usage
-
-When running multiple versions of `node`, we can target
-them directly by asking `n` for the binary path:
+## Using Downloaded Node Versions Without Reinstalling
 
-    $ n bin 0.9.4
-    /usr/local/n/versions/0.9.4/bin/node
+There are three commands for working directly with your downloaded versions of `node`, without reinstalling.
 
-Or by using a specific version through `n`'s `use` sub-command:
+You can show the path to the downloaded version:
 
-    n use 0.9.4 some.js
+    $ n which 6.14.3
+    /usr/local/n/versions/6.14.3/bin/node
 
-Flags also work here:
+Or run a downloaded `node` version with the `n run` command:
 
-    n as 0.9.4 --debug some.js
+    n run 8.11.3 --debug some.js
 
-## Help
+Or execute a command with `PATH` modified so `node` and `npm` will be from the downloaded `node` version.
+(NB: this `npm` will be working with a different and empty global node_modules directory, and you should not install global
+modules this way.)
 
-Output can also be obtained from `n --help`.
+    n exec 10 my-script --fast test
 
-    Usage: n [options/env] [COMMAND] [args]
+## Miscellaneous
 
-    Environments:
-     n [COMMAND] [args]            Uses default env (node)
+Command line help can be obtained from `n --help`.
 
-    Commands:
+List matching remote versions available for download:
 
-      n                              Output versions installed
-      n latest                       Install or activate the latest node release
-      n -a x86 latest                As above but force 32 bit architecture
-      n lts                          Install or activate the latest LTS node release
-      n <version>                    Install node <version>
-      n use <version> [args ...]     Execute node <version> with [args ...]
-      n bin <version>                Output bin path for <version>
-      n rm <version ...>             Remove the given version(s)
-      n prune                        Remove all versions except the active version
-      n --latest                     Output the latest node version available
-      n --lts                        Output the latest LTS node version available
-      n ls                           Output the versions of node available
+    n ls-remote lts
+    n ls-remote latest
+    n lsr 10
+    n --all lsr
 
-    Options:
+List downloaded versions in cache:
 
-      -V, --version   Output version of n
-      -h, --help      Display help information
-      -q, --quiet     Disable curl output (if available)
-      -d, --download  Download only
-      -a, --arch      Override system architecture
+    nvh ls
```

Comment:
`nvh ls` in the README.md is probably meant to be `n ls` for this repo.

(I tried `n ls` and it worked.)

Reply:
Fixed